palladian-commons/pom.xml

```diff
@@ -44,5 +44,27 @@
 			<artifactId>commons-codec</artifactId>
 			<version>1.9</version>
 		</dependency>
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-api-mockito</artifactId>
+			<version>1.6.5</version>
```

Comment:
Use version 1.6.6 as that works better with Jacoco (whether or not they add it).